src/Cody.VisualStudio/CodyPackage.cs

```diff
@@ -86,11 +84,12 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             try
             {
                 InitializeErrorHandling();
+                Configuration.AddFromJsonFile("CodyDevConfig.json");
```

Comment:
The [documentation](https://github.com/sourcegraph/cody-vs/blob/6c4d3cacd9d24369fed756baae6044059668fe0f/CONTRIBUTING.md?plain=1#L51) update is needed, because it has still references to removed `CODY_VS_DEV_PORT` env variable.

Adding additional explanation in the how to set `"CODY_VS_DEV_CONFIG' will be good idea.

Reply:
I've updated the documentation